src/pages/Donation/index.js

```diff
@@ -0,0 +1,82 @@
+import BreadCrumbs from '../../components/NavBreadcrumbs.js';
+import Footer from '../../components/Footer/index.js';
+import Header from '../../components/Header/index.js';
+import React, { useEffect, useState } from 'react';
+import useStyles from './styles.js';
+
+const breadCrumbLinks = [
+  {name: 'Home', href: '/'},
+  {name: 'Radical Collaboration', href: '/donate#root'}, // Update when page is built
+  {name: 'Support Us', href: '/donate'},
+  {name: 'Donate', href: '/donate'},
+];
+
+const Donation = () => {
+  const classes = useStyles();
+  const [windowSize, setWindowSize] = useState(window.innerWidth);
+  useEffect(() => {
+    window.addEventListener('resize', (event) => {
+      setWindowSize(window.innerWidth);
+    });
+  }, []);
+
+  return (
+    <div className={classes.donateContainer}>
+      <Header />
+      <section className={classes.commonSection + ' ' + classes.titleSection}>
+      <BreadCrumbs crumbs={breadCrumbLinks} /> {/* Change left & right padding to 0 */}
+        <h2 className={classes.donateTitle}>Make a Donation</h2>
+      </section>
+
+      <section className={classes.commonSection + ' ' + classes.infoSection}>
+        <div className={classes.infoText}>
+          <p className={classes.infoDescription}>
+            The Civic Tech Index is being developed {windowSize <= 700 ? <br /> : null}
+            by{windowSize > 700 ? <br /> : null}{' '}
+            <span className={classes.styledLink}>Hack For LA,</span>
+            {' '}and is currently taking {windowSize <= 700 ? <br /> : null}
+            donations{windowSize > 700 ? <br /> : null} through{' '}
+            <span className={classes.styledLink}>Code For America.</span>
+          </p>
+
+          <p className={classes.infoSteps}>
+            1. Please follow this link to the {windowSize <= 700 ? <br /> : null}
+             Code For America {windowSize > 700 ? <br /> : null}
```

Comment:
Using the Material-UI grid system will help solve this

Reply:
I just removed the line breaks and allowed the text to naturally wrap. I thought I had to match the Figma exactly but turns out it's just a guidance so that's nice. The cognitive complexity error is gone now.